tests/integration/standard/test_metrics.py

```diff
@@ -146,7 +146,7 @@ def test_unavailable(self):
         # Sometimes this commands continues with the other nodes having not noticed
         # 1 is down, and a Timeout error is returned instead of an Unavailable
         get_node(1).stop(wait=True, wait_other_notice=True)
-
+        time.sleep(5)
```

Comment:
Huh, that's frustrating. Maybe talk to someone to see if there's a possible `ccm` bug here? but for the time being, this is fine.